diff --git a/os_dep.c b/os_dep.c
index 08c501d..49447e0 100644
--- a/os_dep.c
+++ b/os_dep.c
@@ -41,7 +41,7 @@
 #   else /* __GLIBC__ < 2 */
       /* libc5 doesn't have <sigcontext.h>: go directly with the kernel   */
       /* one.  Check LINUX_VERSION_CODE to see which we should reference. */
-#     include <asm/sigcontext.h>
+
 #   endif /* __GLIBC__ < 2 */
 # endif
 #endif /* LINUX && !POWERPC */
